cms/scans.py

```diff
@@ -32,7 +32,7 @@ def parser_selscan_file_conversion(parser=argparse.ArgumentParser()):
             Process a bgzipped-VCF (such as those included in the Phase 3 1000 Genomes release) into a gzip-compressed
             tped file of the sort expected by selscan. 
     """
-    parser.help = """This function converts an input VCF into a TPED file usable by selscan.
```

Comment:
With how `util.cmd.py` is set up, the main description for the command comes from the docstring of the main function it is attached to, so "Process a bgzipped-VCF [...]" will display if the docstring is moved to `main_selscan_file_conversion()`. That will still leave "Undocumented" above the description, which is the help text for the subparser, but we can change `cmd.py` to set that line to the docstring of the parser function (via the subparser help, which means it will also print to the console). So with the proposed change it'll be

    Sub-commands:
    argparse_command_name
    parser_docstring (also printed to console)
    main_function_docstring
    usage: [...]

Unrelated, but I found I had to add a couple flags to force sphinx-build to recreate all docs every time (it wasn't seeing changes for some reason):
`sphinx-build -a -E docs/ ~/Desktop/cms-docs`